_episodes/17-section1-optional-exercises.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Optional exercises for section 1"
+start: false
+teaching: 0
+exercises: 45
+questions:
+- "How can I further finetune my coding environment?"
+objectives:
+- "Explore different options for your coding environment."
+keypoints:
+---
+
+This episode holds some optional exercises for section 1. 
+The exercises have an explorative nature, so see what you like to explore.
+For the remainder of the lesson we recommend sticking with the tools that were introduced in this section.
+
+> ## Exercise: Apply to your own project(s)
+> Apply what you learned in this section to your own project(s). 
+> This is the time to ask any questions to our instructors or let us help you solve problems that you run into.
+> 
+{: .challenge}
+
+> ## Exercise: Try out different Integrated Development Environments
+> Install different Integrated Development Environments (IDEs) and thest them out.
+> Which one do you like the most and why?
+> You can try: 
+> - [Visual Studio Code](https://code.visualstudio.com/)
+> - [Atom](https://atom-editor.cc/)
+> - [Sublime Text](https://www.sublimetext.com/)
+> 
+{: .challenge}
+
+> ## Exercise: Customize the command line
+> You can customize the command line to make yourself more productive.
+> - Try out [z](https://github.com/rupa/z), a simple tool to more quickly move around directories.
+> - Try out [Oh My ZSH](https://ohmyz.sh/), customize the look and feel of your terminal.
+> 
+{: .challenge}
+
+> ## Exercise: Try out different virtual environment managers
+> So far we used `venv`, but there are other virtual environment managers for Python:
+> - [Poetry](https://python-poetry.org/).
+> - [conda (part of Anaconda Distribution)](https://www.anaconda.com/download)
```

Comment:
Given change to Anaconda licensing that make it basically unusable in research, we should probably additionally link to [miniforge](https://github.com/conda-forge/miniforge) saying that it is the tool they should install. The documentation for conda itself is still useful since that basically stays the same.

Reply:
Good point! Attempt to resolve with: 19329a53dfe53810